plugin/db/pg/pg.go

```diff
@@ -697,7 +697,8 @@ func (driver *Driver) FindMigrationHistoryList(ctx context.Context, find *db.Mig
 	}
 	history, err := util.FindMigrationHistoryList(ctx, query, params, driver, find, baseQuery)
 	// TODO(d): remove this block once all existing customers all migrated to semantic versioning.
-	if err != nil {
+	isBytebaseDatabase := strings.Contains(driver.baseDSN, "user=bb") && strings.Contains(driver.baseDSN, "sslmode=disable host=/tmp port=")
```

Comment:
any way we could get the database name "bytebase" and compare it?

Reply:
I think we're using database "bb" and "bbdev" in pg_engine so we're comparing user "bb" here.

Reply:
Let's add a comment?

Reply:
Done.